FreeRTOS/FreeRTOS-Plus-TCP
# [IPv6][SAM] Enable multicast hash table for multiple MAC addresses on SAM network interface

[IPv6][SAM] Enable multicast hash table for multiple MAC addresses on SAM network interface

Description
-----------

The change is made by @htibosch.

When IPv6/Multiple endpoints enabled, the space to store MAC addresses is not enough.
To support it better, use multicast hash table in hardware to reduce loading on space.

Test Steps
-----------

Below scenarios are tested (by @htibosch) on both IPv4/IPv6:
- DNS
- UDP
- TCP
- Ping
- IPERF3

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
